docs/fides/docs/installation/configuration.md

```diff
@@ -10,16 +10,27 @@ After initializing fidesctl, a default configuration file will be generated and
 
 ```toml title="fidesctl.toml"
 [api]
+
+# If FIDESCTL_TEST_MODE env var is set to True, the test_database_url
+# will be used instead of the database URL to avoid overwriting production data
+test_database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl_test"
+
+# The SQLAlchemy connection string used to connect to the database
+# https://docs.sqlalchemy.org/en/14/core/engines.html#postgresql
 database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
-log_destination = ""
-log_level = 20
-log_serialization = ""
+
+# Logging
+log_destination = "" # Also accepts: Any valid file path
+log_level = INFO # Also accepts: TRACE, DEBUG, WARNING, ERROR, CRITICAL
+log_serialization = "" # Also accepts: JSON
 
 [cli]
-server_url = "http://localhost:8080"
+local_mode = False # Tells fidesctl to run without calling a webserver
+server_url = "http://localhost:8080" # The URL of the fidesctl webserver
 
 [user]
-analytics = true
+# The secure key used to encrypt sensitive information in the database
```

Comment:
This feels vague, but I'm not familiar enough with the requirements here to suggest something more helpful to a new user.

Reply:
@SteveDMurphy could you give a more concise description here?

also, what happens if no key is provided? will the values from the db just show up as encrypted or will it fail?

Reply:
I spoke to @SteveDMurphy and dug into the code a bit. I think we should 1) add some validation for presence of an encryption key, and 2) change our default to literally any string. Regardless, using `""` would not break anything; it's used as a PGP key under the hood.